pallets/subtensor/src/swap.rs

```diff
@@ -189,15 +215,15 @@ impl<T: Config> Pallet<T> {
     ///
     /// This function calculates the remaining arbitration period by subtracting the current block number
     /// from the arbitration block number of the coldkey.
-    // pub fn get_remaining_arbitration_period(coldkey: &T::AccountId) -> u64 {
-    //     let current_block: u64 = Self::get_current_block_as_u64();
-    //     let arbitration_block: u64 = ColdkeyArbitrationBlock::<T>::get(coldkey);
-    //     if arbitration_block > current_block {
-    //         arbitration_block.saturating_sub(current_block)
-    //     } else {
-    //         0
-    //     }
-    // }
+    pub fn get_remaining_arbitration_period(coldkey: &T::AccountId) -> u64 {
```

Comment:
This method is still unused, we don't need it.

Reply:
Its a state call , that is still relevant , so I would like to keep it there